docs/API.md

```diff
@@ -121,6 +122,7 @@ tracer.use('pg', {
 * [kafkajs](./interfaces/plugins.kafkajs.html)
 * [knex](./interfaces/plugins.knex.html)
 * [koa](./interfaces/plugins.koa.html)
+* [ldapjs](./interfaces/plugins.ldapjs.html)
```

Comment:
Why? We haven't create any plugin for ldapjs, we have created just a vulnerability analyzer